app/code/core/Mage/Index/Model/Indexer.php

```diff
@@ -284,10 +284,10 @@ public function logEvent(Varien_Object $entity, $entityType, $eventType, $doSave
      * @param   Varien_Object $entity
      * @param   string $entityType
      * @param   string $eventType
-     * @throws Exception
      * @return  Mage_Index_Model_Indexer
+     *@throws Exception|Throwable
```

Comment:
Whitespace missing here and personally I'm no fan of | in the throws statements in phpdocblock. But it's a possibility after all.

Reply:
Fixed whitespace but left open "Exception|Throwable" because I'm unsure if this is consensus.

Reply:
Just an expression of personal opinion. Multiple `throws` annotations give the possibility for a description for each of them after the exception type. If there is no describing text or the same for in this case both `Exception` and `Throwable` then it wouldn't make a difference whether multiple `throws` annotations or | is used. As a bonus I've seen the variant with multiple annotations in multiple places in the codebase ;-).